app/models/item.js

```diff
@@ -166,6 +170,26 @@ export default cloudinaryUrl.extend({
     }
   }),
 
+  hasSingleLocation: Ember.computed('packagesLocations.[]', function(){
+    if(this.get('packagesLocations')){
+      return this.get('packagesLocations').length === 1;
+    }
+  }),
+
+  firstLocationName: Ember.computed('packagesLocations.[]', function(){
+    return this.get('packagesLocations').get('firstObject').get('location.name');
```

Comment:
You can use this `this.get('packagesLocations.firstObject.location.name')` or `this.get('packagesLocations').get('firstObject.location.name')`.
Normally second one is preferred.